packages/tables/resources/views/columns/layout/panel.blade.php

```diff
@@ -1,7 +1,9 @@
-<div @class([
-    'px-4 py-3 bg-gray-100 rounded-lg',
-    'dark:bg-gray-900' => config('forms.dark_mode'),
-])>
+<div
+    {{ $attributes->merge($getExtraAttributes())->class([
```

Comment:
Would it be more readable if all chained methods were on their own line?

Reply:
Agree. I adopted the styling of the other "column" components, e.g. the one of the [TextInput](https://github.com/filamentphp/filament/blob/2.x/packages/tables/resources/views/columns/text-column.blade.php#L10-L39) - there's always a line feed after the div :)

Reply:
Hmm, in other places I've formatted it like I suggested. So should be fine as well if we agree that's more readable.

Reply:
Updated the PR, hopefully I catched it correctly

Reply:
Not entirely, I've commented with a change request. 😄